src/main/java/org/terasology/launcher/updater/LauncherUpdater.java

```diff
@@ -78,6 +78,11 @@ public LauncherUpdater(TerasologyLauncherVersionInfo currentVersionInfo) {
      * @return whether an update is available
      */
     public boolean updateAvailable() {
+        if (this.currentVersionInfo.isEmpty()) {
+            logger.trace("Skipping update check - no version info file found (assuming development environment)");
```

Comment:
I guess that makes sense. When testing the update mechanism locally, a developer can still work around this. For the future, we might want to introduce command line parameters that can steer this behavior.